.github/workflows/ci.yml

```diff
@@ -257,8 +262,9 @@ jobs:
       - name: Test on node.js ${{ matrix.node-version }}
         # Todo(Babel 8): Jest execution path is hardcoded because Yarn 2 does not support node 6
         run: |
-          BABEL_ENV=test node --max-old-space-size=4096 ./node_modules/.bin/jest --ci
```

Comment:
The `--max-old-space-size=4096` was removed. Now that the CI is still green, it is likely that any previous memory issues have already been fixed.